docs/openapi/howto/archboard_feedback.md

```diff
@@ -0,0 +1,152 @@
+
+Content:
+- [Need for more/less SDK package(s)](#need-for-moreless-sdk-packages)
+- [Rename an operation](#rename-an-operation)
+- [Rename an operation group (or rename a sub-client)](#rename-an-operation-group-or-rename-a-sub-client)
+- [Need one client to have all operations directly on it (no sub-client)](#need-one-client-to-have-all-operations-directly-on-it-no-sub-client)
+- [Need one client/builder with sub-clients](#need-one-clientbuilder-with-sub-clients)
+- [Need two or more service clients (regardless of if some needs subclients or not)](#need-two-or-more-service-clients-regardless-of-if-some-needs-subclients-or-not)
+- [Rename a model name](#rename-a-model-name)
+- [Rename a model attribute](#rename-a-model-attribute)
+- [Operation should be pageable](#operation-should-be-pageable)
+
+## Need for more/less SDK package(s)
+
+If you got as a feedback that your API needs to split across several packages, or two packages should be merged into one, then
+structure your Swaggers to have exactly one folder per package. Most of the work is re-organizing files and Readme. Example of hierarchy on 
+the [RestAPI spec repo](https://github.com/Azure/azure-rest-api-specs):
+
+```
+specification/
+├─ myservice/
+│  ├─ data-plane/
+│  │  ├─ package1/
+│  │  │  ├─ Microsoft.MyService/ (swaggers for package 1)
+│  │  │  ├─ readme.md
+│  │  ├─ package2/
+│  │  │  ├─ Microsoft.MyService/ (swaggers for package 2)
+│  │  │  ├─ readme.md
+```
+
+## Rename an operation
+
+Operation name are directly OperationID in Swagger. Just rename the OperationID directly.
+
+Note: If your operation contains an `_`, just change the second part as the first is an operation group.
+
+``` json
+ "operationId": "SomethingToDoOperation",
+ ...
+ "operationId": "MyGroup_SomethingElseToDoOperation",
+```
+becomes
+``` json
+ "operationId": "BuildTheThing",
+ ...
+ "operationId": "MyGroup_BuildSomethingElse",
+```
+
+## Rename an operation group (or rename a sub-client)
+
+Operation groups are the first part of an OperationID in Swagger, before the `_` character. To rename an operation group, you need to rename all OperationID that
+starts with the operation group prefix (there is likely many of them).
+
+``` json
+ "operationId": "MyGroup_BuildSomethingElse",
+```
+becomes
+``` json
+ "operationId": "NewGroup_BuildSomethingElse",
+```
+
+## Need one client to have all operations directly on it (no sub-client)
+
+Don't use operation groups in your OperationID (remove them if they exist). For any operation with an `_` character, remove the first part.
+
+``` json
+ "operationId": "MyGroup_BuildSomethingElse",
+```
+becomes
+``` json
+ "operationId": "BuildSomethingElse",
+```
+
+## Need one client/builder with sub-clients
+
+-	Use operation groups, and name them as you want your subclients to be called. Operation group can be create in Swagger by using the `_` character in the OperationID 
+(`OperationGroupName_OperationName`)
+-	For C#, use `single-top-level-client: true`
```

Comment:
Would all languages need to support this capability?

Reply:
All languages should be able to create a single client by default, or support a flag to do so.